src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Marshalling/ComObject.cs

```diff
@@ -135,5 +147,10 @@ VirtualMethodTableInfo IUnmanagedVirtualMethodTableProvider.GetVirtualMethodTabl
 
             return new(result.ThisPtr, result.Table);
         }
+
+        object ComImportInteropInterfaceDetailsStrategy.IComImportAdapter.GetRuntimeCallableWrapper()
+        {
+            return _runtimeCallableWrapper!;
```

Comment:
```suggestion
            Debug.Assert(_runtimeCallableWrapper != null);
            return _runtimeCallableWrapper;
```
Is this guaranteed to be non-zero when this method is called? If it is the case, `Debug.Assert` would be better than `!`.